### PR TITLE
Feature/toddys examples

### PR DESCRIPTION
**Pull request by toddysm**
Examples of the automation scripts and the environment variables for DB access

### CODE REVIEW

File: example/functions-example/HttpTriggerToddysmExample/function.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
+{
+  "scriptFile": "__init__.py",
+  "bindings": [
+    {
+      "authLevel": "anonymous",
+      "type": "httpTrigger",
+      "direction": "in",
+      "name": "req",
+      "methods": [
```
Comment (nataliagourova) [inline example/functions-example/HttpTriggerToddysmExample/function.json:9]: adding "delete" would be helpful

**Review by nataliagourova — APPROVED** [review]

File: example/functions-example/HttpTriggerToddysmExample/function.json (hunk 1 above)
Comment (theKunte) [reply example/functions-example/HttpTriggerToddysmExample/function.json:9]: would we add patch? Just a question in regards to last quarter for the different responses.

Comment (toddysm) [reply example/functions-example/HttpTriggerToddysmExample/function.json:9]: @nataliagourova and @theKunte depends on what endpoint this function implements. If it is only the `/users` you need to think whether `PUT/PATCH` and `DELETE` are appropriate.

File: example/functions-example/HttpTriggerToddysmExample/__init__.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
+import logging
+import os
+
+import azure.functions as func
+
+
+def main(req: func.HttpRequest) -> func.HttpResponse:
+    logging.info('Python HTTP trigger function processed a request.')
+
+    db_username = os.environ["ENV_DATABASE_USERNAME"]
```
[the remark this answers is not in the document]
Comment (toddysm) [reply example/functions-example/HttpTriggerToddysmExample/__init__.py:10]: Not sure what you mean with *"we didn't actually run the endpoint url in the demo to see the result"* - can you clarify?

[the remark this answers is not in the document]
Comment (farhadbahrehmandhenry) [reply example/functions-example/HttpTriggerToddysmExample/__init__.py:10]: double checked the lecture, I was wrong. however I was able to access the env variables by putting them under "values" in local.settings.json and getting them by importing "os" and calling os.environ[], exactly like what you had done in the lecture. just wanted to confirm it's working properly.

**Review by farhadbahrehmandhenry — APPROVED** [review]

File: example/automation-example/function-automation/deploy-template.ps1 [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+    [Parameter(Mandatory=$True)]
+    [string]
+    $ServicePrincipalId,
+   
+    [Parameter(Mandatory=$True)]
+    [string]
+    $ServicePrincipalPassword
+)
+
+Connect-AzAccount
```
Comment (toddysm) [inline example/automation-example/function-automation/deploy-template.ps1:19]: Can you change something here?

Comment (farhadbahrehmandhenry) [reply example/automation-example/function-automation/deploy-template.ps1]: I believe this is good to merge!

**Pull request merged**